CMakeLists.txt

```diff
@@ -21,6 +21,10 @@ set(mgclient_SOVERSION 2)
 add_definitions(-DMGCLIENT_VERSION="${mgclient_VERSION}")
 
 # Deal with the operating system.
+if((NOT UNIX) AND EMSCRIPTEN)
+    message(FATAL_ERROR "WASM build is only supported in Linux")
```

Comment:
Aha, so, we'll build `webasm` only on Linux and then run it elsewhere?

Reply:
Exactly!